airflow_hop/xml.py

```diff
@@ -127,11 +143,27 @@ def __get_pipe_parameters(self, pipeline_file) -> Element:
         tree_root = tree.getroot()
         parameters = tree_root[0].findall('parameters')
         root = Element('parameters')
+        
+        task_param_names = set(self.task_params.keys())
+
+        # Add existing parameters from the workflow
```

Comment:
Shouldn't this say "Add existing parameters from the pipeline"?

Reply:
Yes that should. I can edit that now.